hardware_interface/test/joint_limits/joint_limits_rosparam_test.cpp

```diff
@@ -28,7 +28,8 @@ class JointLimitsRosParamTest : public ::testing::Test
   void SetUp()
   {
     rclcpp::NodeOptions node_options;
-    node_options.allow_undeclared_parameters(true)
+    node_options
+    .allow_undeclared_parameters(true)
```

Comment:
Why new line?

Reply:
It is using method chaining. Notice no semicolon on the end of line 32....there is another .method() call below it. Read better to me to have the multiple .method() chain calls indented to match and it passed clint/uncrustify.